src/components/Assignments.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState, useEffect} from 'react';
+import AssignmentService from '../services/assignmentservice';
+import CreateIcon from '@material-ui/icons/Create';
+import DeleteIcon from '@material-ui/icons/Delete';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import AssignemntForm from './AssignmentForm';
+
+
+function Assignments() {
+ 
+
+    const [assignment, setAssignment] = useState([])
+    const [currentId, setCurrentId] = useState('')
+    const [open, setOpen] = useState(false);
+    
+    useEffect(() => {
+      AssignmentService.getAssignment().then((Assignment) => {
+        setAssignment(assingment);
+      });
+    }, []);
+    
+    const handleClickOpen = () => {
+   setOpen(true);
```

Comment:
the inconsistent indentation in this file makes is unnecessarily difficult to read and understand what's going on, this line for example is unindented compared to the line above when I'd expect to be further indented
in other places in this file there's a mix between things being indented with one space vs. two spaces (vs. I suspect tabs)